config/locales/patient_session_statuses.en.yml

```diff
@@ -34,7 +34,7 @@ en:
       colour: red
       text: Could not vaccinate
       banner_title: Could not vaccinate
-      banner_explanation: "%{triage_nurse} that %{full_name} should not be vaccinated"
+      banner_explanation: "%{triage_nurse} decided that %{full_name} should not be vaccinated"
```

Comment:
> However, I'm not convinced this message is actually shown anywhere. This state is an outcome state and the banner explanation may not be shown at all.

Maybe delete this and set [raise_on_missing_translations](https://api.rubyonrails.org/v7.0.0/classes/ActionView/Helpers/TranslationHelper.html)? Apparently that was [recently changed to cover more cases of the missing translation being used](https://github.com/rails/rails/pull/47105).

Reply:
Oh yeah, that's a really good idea... Is there any reason not to enable it generally in dev and test? https://github.com/nhsuk/manage-childrens-vaccinations/pull/1158

Reply:
So:
* I enabled the `raise_on_missing_translations` locally in dev and test
* removed the string
* re-ran the tests and nothing broke

So I've now yanked that line, instead of fixing the typo. I will do another clean-up of the strings in that translation file in a follow-up PR.